ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -15,6 +15,13 @@ import com.revenuecat.purchases.paywalls.PaywallData
 import java.net.URL
 
 internal object TestData {
+    const val title = "Call to action for better conversion."
+    const val subtitle = "Lorem ipsum is simply dummy text of the printing and typesetting industry."
+    const val callToAction = "Subscribe for {{ price_per_period }}"
+    const val offerDetails = "{{ total_price_and_per_month }}"
+    const val offerDetailsWithIntroOffer = "{{ total_price_and_per_month }} after {{ sub_offer_duration }} trial"
+    const val offerName = "{{ sub_period }}"
```

Comment:
These will be different for the different templates. Could the test access `TestData.template2.localization` instead?

Reply:
Hmm right... As I mentioned, that might work for now, but once we actually perform the variable replacement, we will need to hardcode the expected strings in the tests, not using the original ones. But well, I can do that for now and fix the tests once we do the variable replacement 👍